Source/Core/DolphinWX/Input/InputConfigDiag.cpp

```diff
@@ -144,7 +144,8 @@ void PadSettingExtension::UpdateValue()
 
 PadSettingCheckBox::PadSettingCheckBox(wxWindow* const parent,
                                        ControllerEmu::ControlGroup::BooleanSetting* const _setting)
-    : PadSetting(new wxCheckBox(parent, wxID_ANY, wxGetTranslation(StrToWxStr(_setting->m_name)))),
+    : PadSetting(
+          new wxCheckBox(parent, wxID_ANY, wxGetTranslation(StrToWxStr(_setting->m_ui_name)))),
```

Comment:
The line is a bit longer now, and that's how clang-format wants to format it now.